.github/SUPPORT.md

```diff
@@ -0,0 +1,25 @@
+# Support
+
+Node.js contributors have limited availability to address general support
+questions. Please make sure you are using a [currently-supported version of
+Node.js](https://github.com/nodejs/Release#release-schedule).
+
+When looking for support, please first search for your question in these venues:
+
+* [Node.js Website](https://nodejs.org/en/)
```

Comment:
Suggestion based on @bnb's comment about including API docs:

```suggestion
* [Node.js Website](https://nodejs.org/en/), especially the
  [API docs](https://nodejs.org/en/docs/)
```

Normally, I'd go with _API documentation_ rather than _API docs_, but the website itself calls them _docs_ so might as well go with the flow.